state.go

```diff
@@ -824,3 +828,11 @@ func (o *OverflowState) Parse(codeFileName string, code []byte, network string)
 
 	return strings.TrimSpace(string(program2.Code())), nil
 }
+
+func (o *OverflowState) GetCoverageReport() *runtime.CoverageReport {
+	return o.Emulator.GetCoverageReport()
```

Comment:
This will not be necessary when we are able to inject a `CoverageReport` with the options API of the emulator, e.g

```go
coverageReport := runtime.NewCoverageReport()
emulator.WithCoverageRepor(coverageReport)
```

This is not yet available unfortunately..

What do you think about this approach?

Reply:
So you mean create an object and send in a pointer to it?

Reply:
Yes, exactly. Under the hood, it will be passed down to the `runtime.Config` [object](https://github.com/onflow/cadence/blob/master/runtime/coverage_test.go#L1233-L1246).

Reply:
Yeah that makes sense.